tests/test_trnsys.py

```diff
@@ -1,16 +1,25 @@
+import math
 from pathlib import Path
 
 import pytest
 
 
 from trnpy.trnsys.simulation import Simulation
 from trnpy.trnsys.lib import TrnsysLib, TrnsysDirectories, track_lib_path
-from trnpy.exceptions import DuplicateLibraryError, TrnsysStepForwardError
+from trnpy.exceptions import (
+    DuplicateLibraryError,
+    TrnsysStepForwardError,
+    TrnsysGetOutputValueError,
+    TrnsysSetInputValueError,
+)
 
 
 class MockTrnsysLib(TrnsysLib):
     def __init__(self):
         self.is_at_end = False
+        self.units = set()  # tracks units we want to be present in the deck
```

Comment:
Nit: Let's avoid inline comments whenever possible. It's best to document all public attributes in the docstring for the class. We're using the Google style Python docs, and I'm sure their style guide has an example of how this should be done.

Reply:
I agree - the `MockTrnsysLib` class needs some work (including better documentation) to make it more useful.